server/service/apple_mdm.go

```diff
@@ -1498,7 +1495,7 @@ func (svc *Service) BatchSetMDMAppleProfiles(ctx context.Context, tmID *uint, tm
 		return ctxerr.Wrap(ctx, err)
 	}
 
-	appCfg, err := svc.AppConfigObfuscated(ctx)
+	appCfg, err := svc.ds.AppConfig(ctx)
```

Comment:
The code looks good only that I do not understand this last change.
Could you please add some notes here?

Reply:
It's loading the application config from the datastore directly (`svc.ds.AppConfig`), instead of via the service abstraction (`svc. AppConfigObfuscated`) because the latter checks that the user can read app config (which we are not returning to the GitOps user so we should be a-ok, this is loading the config to check for the `EnabledAndConfigured` field).

Reply:
Understood.
I would kindly ask you to write a section of test plan where you think of all the cases that this will efect.

Here is an example but could be any format you'd like:
[ ] Test read by admin - should pass - covered by unit tests: YES.
[ ] Test read by GitOps - should fail - covered by unit tests: NO (Need manual tests).
...
[ ] Test write by ...

Reply:
If existing, you can refer to a unit test file that has all those tests instead.

Reply:
> Here is an example but could be any format you'd like:

Yes. This is all tested in https://github.com/fleetdm/fleet/blob/main/server/authz/policy_test.go.
E.g. for testing application config access control: https://github.com/fleetdm/fleet/blob/f53a896f09cf826d0ebd9eddcf798676c69b60c6/server/authz/policy_test.go#L44-L85